### PR TITLE
Adaption to aldryn-newsblog

### PR DESCRIPTION
**Pull request by FinalAngel**
- cleanup README
- fix setup.py to be aligned with https://github.com/aldryn/aldryn-newsblog/blob/master/setup.py

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -1,39 +1,41 @@
...
+A Plugin for django CMS to add CSS classes to other plugins.
 
+------------
 Installation
 ------------
 
 This plugin requires `django CMS` 2.4 or higher to be properly installed.
 
-* In your projects `virtualenv`_, run ``pip install aldryn-style``.
+* Create a `virtualenv` and run ``pip install aldryn-style``
```
Comment (Chive) [inline README.rst]: You need to use double quotes in order to mark text as inline code

File: setup.py [hunk 2]
```diff
@@ -16,11 +12,9 @@
     'License :: OSI Approved :: BSD License',
     'Operating System :: OS Independent',
     'Programming Language :: Python',
-    'Topic :: Communications',
     'Topic :: Internet :: WWW/HTTP :: Dynamic Content',
-    'Topic :: Internet :: WWW/HTTP :: Dynamic Content :: Message Boards',
-    'Programming Language :: Python :: 2.6',
-    'Programming Language :: Python :: 2.7',
```
Comment (Chive) [inline setup.py]: Don't remove this

File: setup.py [hunk 3]
```diff
@@ -30,12 +24,11 @@
...
     author_email='info@divio.ch',
     url='https://github.com/divio/aldryn-style',
-    packages=['aldryn_style', 'aldryn_style.migrations'],
-    install_requires=INSTALL_REQUIRES,
+    packages=find_packages(),
     license='LICENSE.txt',
     platforms=['OS Independent'],
+    install_requires=REQUIREMENTS,
     classifiers=CLASSIFIERS,
-    long_description=open('README.md').read(),
```
Comment (Chive) [inline setup.py]: neither remove this

File: setup.py (hunk 2 above)
Comment (FinalAngel) [inline setup.py]: but this is not added within aldryn-newsblog @Chive

File: README.rst [hunk 4]
```diff
@@ -1,39 +1,41 @@
...
 ```
+gettext = lambda s: s
 ALDRYN_STYLE_CLASS_NAMES = (
-    ('info', _("info")),
-    ('new', _("new")),
-    ('hint', _("hint")),
+    ('info', gettext('info')),
+    ('new', gettext('new')),
+    ('hint', gettext('hint')),
 )
```
Comment (Chive) [inline README.rst]: why did you change this? btw you messed up the formatting

File: setup.py (hunk 3 above)
Comment (FinalAngel) [inline setup.py]: also not like this within aldryn-newsblog /cc @Marketionist

File: setup.py (hunk 2 above)
Comment (Chive) [inline setup.py]: It should be there then as well.

File: README.rst (hunk 4 above)
Comment (FinalAngel) [inline README.rst]: this only would work in our standard setup not if you'd use ``gettext = lambda s: s`` which the tutorials reference to

File: setup.py (hunk 3 above)
Comment (Chive) [inline setup.py]: It should be there as well then. Otherwise you have no README on pypi.

Comment (FinalAngel) [inline setup.py]: yeah will need to know whats @mkoistinen approach is there, as it's missing and not streamlined across the addons

File: README.rst (hunk 4 above)
Comment (Chive) [inline README.rst]: Using ``_`` for any ``gettext`` function is a coding convention defined by Django (https://docs.djangoproject.com/en/1.8/topics/i18n/translation/#standard-translation). I see no issue with that there. And I'd rather prefer to have the tutorial changed than this app.

File: setup.py (hunk 3 above)
Comment (Chive) [inline setup.py]: Apparently pypi changed it's README detection mechanism? @mkoistinen any idea?

File: README.rst (hunk 4 above)
Comment (FinalAngel) [inline README.rst]: the generated setting from django 1.7 adds this: ``gettext = lambda s: s`` check: https://github.com/divio/django-cms-demo/blob/master/src/settings.py#L13

Comment (Chive) [inline README.rst]: Sorry, but that is just plain not true. https://github.com/django/django/blob/stable/1.7.x/django/conf/project_template/project_name/settings.py

Comment (FinalAngel) [inline README.rst]: where the **** did that come from than - ``_ = lambda s: s`` is than the convention @Chive ?

File: setup.py (hunk 3 above)
Comment (mkoistinen) [inline setup.py]: Definitely want to keep these items, @FinalAngel and looks like we need to add them to NewBlog too.

File: README.rst (hunk 4 above)
Comment (Chive) [inline README.rst]: I don't know.

Comment (mkoistinen) [inline README.rst]: Its a bit more than a just a convention. The GetText tools parse the source files and look for the gettext functions by name OR the "_". The only reason we have to do anything different in the settings.py file is that we don't (want to) import modules there, so doing `from django.utils.translations import ugettext as _` is out of the question. Using the lambda makes it pass Python parsers, and the GetText parser will then still pick it up.

I actually agree with the lambda definition here, @chive, as it makes it clear what the intent is.

File: setup.py (hunk 2 above)
Comment (mkoistinen) [inline setup.py]: It should be in NewsBlog too, I agree with @chive here.

File: setup.py [hunk 5]
```diff
@@ -16,11 +12,12 @@
     'License :: OSI Approved :: BSD License',
     'Operating System :: OS Independent',
     'Programming Language :: Python',
-    'Topic :: Communications',
-    'Topic :: Internet :: WWW/HTTP :: Dynamic Content',
-    'Topic :: Internet :: WWW/HTTP :: Dynamic Content :: Message Boards',
     'Programming Language :: Python :: 2.6',
     'Programming Language :: Python :: 2.7',
+    'Topic :: Internet :: WWW/HTTP :: Dynamic Content',
+    'Topic :: Internet :: WWW/HTTP :: Dynamic Content :: Message Boards',
```
Comment (Chive) [inline setup.py]: You can remove ``Topic :: Internet :: WWW/HTTP :: Dynamic Content :: Message Boards`` since it makes no sense

File: setup.py [hunk 6]
```diff
@@ -16,11 +12,12 @@
...
+    'Topic :: Software Development',
+    'Topic :: Software Development :: Libraries :: Application Frameworks',
```
Comment (Chive) [inline setup.py]: remove this as well ``Topic :: Software Development :: Libraries :: Application Frameworks``

**Comment by Chive** [comment]
LGTM!
